buildkite/src/Jobs/Release/MinaToolchainArtifactFocal.dhall

```diff
@@ -1,38 +1,19 @@
-let S = ../../Lib/SelectFiles.dhall
+let ArtifactPipelines = ../../Command/MinaArtifact.dhall
 
 let Pipeline = ../../Pipeline/Dsl.dhall
 
 let PipelineTag = ../../Pipeline/Tag.dhall
 
-let JobSpec = ../../Pipeline/JobSpec.dhall
+let DebianVersions = ../../Constants/DebianVersions.dhall
 
-let DockerImage = ../../Command/DockerImage.dhall
+let Artifacts = ../../Constants/Artifacts.dhall
 
 in  Pipeline.build
-      Pipeline.Config::{
```

Comment:
Toolchain build is no longer in PR, but we have a separate pipeline for this kind of build. Therefore we can freely ignore missing dirty when